### DISCUSSION TITLE
How to remove a long-dead / thrown away device from network?

### DISCUSSION THREAD

**Discussion by decairn** [zwave-js/zwave-js#6832] [Q&A] [answered]
My z-wave network is a bit littered with old and now failed devices some of which were thrown away ages ago.  How to remove these from the network entirely?

**Comment by AlCalzone** [comment, marked as answer]
Using the **remove failed device** feature. Should be in the **advanced** menu in Z-Wave JS UI.

**Comment by decairn** [comment]
All good, thank you.
